modules/api/src/main/java/org/eclipse/xpanse/api/AdminServicesApi.java

```diff
@@ -46,6 +49,8 @@
 @Secured({ROLE_ADMIN})
 public class AdminServicesApi {
 
+    private final TerraformApi terraformApi = new TerraformApi();
```

Comment:
@jinyangyang222 I see there is some issue with the generated classes. The generated classes must be spring beans and simply be `autowired`. 
<generatorName>java</generatorName> -> must be `spring` for the classes to be generated as spring beans.

Can you please check this?

Reply:
ok,I'll check this.